dataeng/jobs/analytics/SnowflakeMicrobachelorsITK.groovy

```diff
@@ -49,7 +48,14 @@ class SnowflakeMicrobachelorsITK {
             wrappers {
                 timestamps()
             }
-            publishers common_publishers(allVars)
+            publishers {
+                archiveArtifacts {
+                    // job normally will generate two unique CSV files, each ending with timestamps as part of the file name
+                    pattern('*.csv')
```

Comment:
When I created the job, it provided this warning:  

  | ‘*.csv’ doesn’t match anything, but ‘analytics-tools/snowflake/*.csv’ does. Perhaps that’s what you mean?

Reply:
Ah, yeah. A misunderstanding from my end. Looks like I made a bad assumption from where the archive step action would occur since the bash script changes directory into the analytics-tools/snowflake directory when executed. 

I'll push an update and thanks again for your help and patience!